### DISCUSSION TITLE
Adding ngrok ingress controller support

### DISCUSSION THREAD

**Discussion by shalomy-cyera** [strimzi/strimzi-kafka-operator#9150] [Ideas]
Hi,
Is adding the [ngrok ingress controller](https://github.com/ngrok/kubernetes-ingress-controller) support on the road map? If not, is there some way to add it or point me on how to add this support?

Thanks

**Comment by scholzj** [comment]
No, there are no such plans. Sorry.

I guess you have two options:
* You can simply try if it works. Strimzi is developed and tested with the Kubernetes Nginx Ingress Controller. That does not mean it does not work with any other. The different Ingress controllers are not 100% compatible. So it might work or it might not. You would need to try it. A prerequisite is TLS passthrough support.
* You can set it up yourself:
  1. Configure a `type: cluster-ip` service in Strimzi
  2. Create your Ingres resources manually for each broker
  3. In the Kafka CR, configure the _advertised host and port_ with the addresses matching your Ingress resources.

**Reply by shalomy-cyera** [reply]
Ok, thanks for the guidance!
